src/System.Runtime.WindowsRuntime.UI.Xaml/src/System.Runtime.WindowsRuntime.UI.Xaml.csproj

```diff
@@ -2,11 +2,16 @@
   <PropertyGroup>
     <AssemblyName>System.Runtime.WindowsRuntime.UI.Xaml</AssemblyName>
     <ProjectGuid>{263DA4F1-C3BC-4B43-98E7-9F38B419A131}</ProjectGuid>
-    <PackageTargetRuntime>win8</PackageTargetRuntime>
-    <Configurations>uap-Windows_NT-Debug;uap-Windows_NT-Release</Configurations>
+    <Configurations>netcoreapp-Windows_NT-Debug;netcoreapp-Windows_NT-Release;netstandard-Debug;netstandard-Release;netstandard1.1-Debug;netstandard1.1-Release;uap-Windows_NT-Debug;uap-Windows_NT-Release;uapaot-Windows_NT-Debug;uapaot-Windows_NT-Release</Configurations>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetGroup)' == 'netstandard1.1' or '$(TargetGroup)' == 'netstandard'">
+    <GeneratePlatformNotSupportedAssembly>true</GeneratePlatformNotSupportedAssembly>
+    <!--<GeneratePlatformNotSupportedAssemblyMessage>SR.PlatformNotSupported_WindowsRuntimeUIXaml</GeneratePlatformNotSupportedAssemblyMessage>-->
```

Comment:
I think my fix went in for this. https://github.com/dotnet/buildtools/commit/476f7aee0d8e80dfa379605dc6265c8af36b9f24

Reply:
I use https://github.com/dotnet/corefx/pull/31684 to test whether your fix is ported to corefx. if it is ported, then I will update this.